frontend/src/global_styles/openproject/_forms.sass

```diff
@@ -157,6 +156,9 @@ select
 
   &:not(.FormControl-select)
     @if $select-arrow
-      background: transparent url(image-triangle($select-arrow-color)) right 10px center no-repeat
```

Comment:
Please remove the `image-triangle` function as well as it is not used anywhere else